pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -1806,9 +1806,9 @@ statfunc u32 cgroup_skb_submit(void *map, struct __sk_buff *ctx, net_event_conte
 #define retval_hasflag(flag) (neteventctx->eventctx.retval & flag) == flag
 
 SEC("cgroup/sock_create")
-int cgroup_sock_create(struct bpf_sock *ctx)
+int cgroup_sock_create(struct bpf_sock *sk)
```

Comment:
I can revert this change but this make clearer in `bpf_sk_storage_get` what we are passing as 2nd argument